tests/rule_based_profiler/parameter_builder/test_parameter_computations.py

```diff
@@ -150,108 +148,3 @@ def test_bootstrap_point_estimate_efficacy(
             atol=EFFICACY_TOLERANCE,
             err_msg=f"Actual value of {actual_false_positive_rates[distribution]} differs from expected value of {false_positive_rate} by more than {ATOL + EFFICACY_TOLERANCE * abs(actual_false_positive_rates[distribution])} tolerance.",
         )
-
-
-def test_bootstrap_point_estimate_bias_corrected_efficacy(
```

Comment:
Is this test no longer relevant / necessary?

Reply:
@anthonyburdi @NathanFarmer I found that all 3 (three) tests used methods that no longer existed anywhere!  One of them could be resurrected by removing the underscore from the method's name (which is what I did).  @NathanFarmer could you please shed some light on this?  I could not even run `pytest` with those broken imports.  Thanks.

Reply:
I removed the broken/unnecessary pieces in PR #4818. But why didn't CI/CD catch this in PR #4742? I wonder if we are having problems with `dgtest`.

Reply:
Thanks, @NathanFarmer -- the merge was smooth.